src/entities/Resource.js

```diff
@@ -40,6 +41,9 @@ export default class Resource {
    */
   setDesc(value) {
     this.desc = value;
+    if (value !== undefined) {
+      this.desc = marked(value).replace(/<p>|<\/p>/gi, '');
```

Comment:
Can move it from entity into service ? I think entity shouldn't add some transformation with using third-party libraries.

Reply:
Agree with @veeeeeRySeXy, that it shouldn't be here.
That's a responsibility of presentation layer (VueJS templates), not entity, and even not service (logic layer).
Ideally I would see it as a [VueJS Filter](https://vuejs.org/v2/guide/filters.html) in template.

Reply:
@sariluong see an answer [here](https://stackoverflow.com/questions/12429729/controller-belongs-to-the-presentation-layer) that explains what is presentation layer.
Model/Entity or Logic layers **should not** take care of how data looks. But presentation layer (in this case - VueJS template, which is part of View in MVC) **must** take care of it.

Reply:
I agree. Maybe, I will move it to a mixin and reuse at TutorialItem, PackageItem and WebsiteItem.